app/views/layouts/all-components.njk

```diff
@@ -22,6 +22,6 @@
   {% for data in componentData %}
     {% set componentName = data.componentName %}
     {% set componentNameHuman = componentName | replace("-", " ") %}
-    {{ showExamples(componentName, componentNameHuman, data) }}
+    {{ showExamples(componentName, componentNameHuman, data, legacy_query) }}
```

Comment:
We seem to use camel case pretty consistently for view variables – should we update this to match?